.coveragerc

```diff
@@ -0,0 +1,23 @@
+[run]
```

Comment:
I wonder if we want to set `branch = True` here?
https://coverage.readthedocs.io/en/6.5.0/branch.html#branch

Reply:
@sidsatyelp branch coverage is a good idea. I originally had that here but it significantly increase the amount of information that gets reported so I elected to leave it out and consider adding it in the future. Having said that, I'm happy to add branch coverage right now if you prefer!

Reply:
If that is the case then we could leave it out for now. When there is too much information reported, people tend to skip it! We can add it later if we see the need for it.

Reply:
For posterity, coverage report [with --branch](https://fluffy.yelpcorp.com/i/Jd2WmXCL9jp97GcKGtl17hr48pqlXjDg.html) and [without --branch](https://fluffy.yelpcorp.com/i/9qcP35c3s0q9N5M2BJnFK7S2Dlm2Ln6f.html)